mycroft/audio/services/simple/__init__.py

```diff
@@ -20,17 +20,22 @@
 from mycroft.util.log import LOG
 from mycroft.util import play_mp3, play_ogg, play_wav
 import mimetypes
+import re
 from requests import Session
 
 
 def find_mime(path):
     mime = None
     if path.startswith('http'):
         response = Session().head(path, allow_redirects=True)
+        print(response.status_code)
```

Comment:
Is this just for debugging?  Should you log instead?

Reply:
It should be removed. Good catch

Reply:
Commit updated.